types/helmet/index.d.ts

```diff
@@ -105,8 +105,12 @@ declare namespace helmet {
         'worker-src'?: HelmetCspDirectiveValue;
     }
 
+    export interface IHelmetContentSecurityReportOnlyFunction {
+        (req: express.Request, res: express.Response): boolean;
+    }
+
     export interface IHelmetContentSecurityPolicyConfiguration {
-        reportOnly?: boolean;
+        reportOnly?: boolean | IHelmetContentSecurityReportOnlyFunction;
```

Comment:
Rather than declare a whole interface for this, you could probably just do the following since it's not re-used anywhere.

```
reportOnly?: boolean | (req: express.Request, res: express.Response): boolean;
```

Reply:
This fails here. Needs to be declared with arrow style and enclosed in parens as it seems. Or am I missing something?

```reportOnly?: boolean | ((req: express.Request, res: express.Response) => boolean);```